examples-gallery/plot_betts_10_103_104.py

```diff
@@ -228,15 +178,14 @@
 solution, info = prob.solve(initial_guess)
 print(info['status_msg'])
 if info['obj_val'] < 12.8738850:
-    msg = 'so, opty with ODE gets a better result'
+    msg = 'opty with ODE gets a better result'
 else:
     msg = 'opty with ODE gets a worse result'
 print(f'Minimal objective value achieved: {info['obj_val']:.4f},  ' +
-            f'as per the book it is {12.8738850}, {msg} ')
+      f'as per the book it is {12.8738850}, {msg} ')
 time_ODE = time.time() - start
 print(f'Time taken for the simulation: {time_ODE:.2f} s')
```

Comment:
I don't think these timings are fair comparisons. You should only time the call for `solve()` to compare and you can also do that by pulling the values from IPOPT or just counting the number of function iterations, also available for IPOPT. I may remove the timing.